### PR TITLE
wine 9.12

### PR DESCRIPTION
**Pull request by km9l**
~~didnt build, test requested~~

### CODE REVIEW

File: segregrevert.patch [hunk 1]
```diff
@@ -66,22 +72,31 @@ index 771f32bb514..76ad21f6be5 100644
...
          todo_wine ok( context->SegFs && context->SegFs != context->SegSs,
                        "got fs %#x\n", context->SegFs );
-@@ -3055,12 +3055,12 @@ static void test_exceptions(void)
-         ok( ctx.SegDs == ds, "wrong ds %04x / %04x\n", ctx.SegDs, ds );
-         ok( ctx.SegEs == es, "wrong es %04x / %04x\n", ctx.SegEs, es );
+@@ -3062,10 +3065,12 @@ static void test_exceptions(void)
          ok( ctx.SegFs == fs, "wrong fs %04x / %04x\n", ctx.SegFs, fs );
--        ok( ctx.SegGs == gs || !gs, "wrong gs %04x / %04x\n", ctx.SegGs, gs );
-+        ok( ctx.SegGs == gs, "wrong gs %04x / %04x\n", ctx.SegGs, gs );
+         ok( ctx.SegGs == gs || !gs, "wrong gs %04x / %04x\n", ctx.SegGs, gs );
```
Comment (km9l) [inline segregrevert.patch:77]: i may have missed this..

**Comment by km9l** [comment]
tested, thanks @HoodB0
(studio is still awful without childwindow...)

**Comment by apprehensions** [comment]
childwindow is in wine

**Comment by km9l** [comment]
> childwindow is in wine
its unfinished

the patch i just pushed makes it work somewhat, but its reliability depends a lot on renderer and the gpu
we need more testing

**Pull request closed without merging**